### PR TITLE
Enable Interactive JS ROOT Graphics in JupyterLab

### PR DESCRIPTION
**Pull request by philipleindecker**
Add requirejs and update _getUID() function

### CODE REVIEW

File: bindings/pyroot/JupyROOT/helpers/utils.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 <div id="{jsDivId}"
      style="width: {jsCanvasWidth}px; height: {jsCanvasHeight}px">
 </div>
-
+<script src="/static/components/requirejs/require.js?v=951f856e81496aaeec2e71a1c2c0d51f" type="text/javascript" charset="utf-8"></script>
```
Comment (etejedor) [inline bindings/pyroot/JupyROOT/helpers/utils.py]: What is the `v` variable that is being set here? Is it strictly required?

File: bindings/pyroot/JupyROOT/helpers/utils.py [hunk 2]
```diff
@@ -375,8 +374,7 @@ def _getUID(self):
         Every DIV containing a JavaScript snippet must be unique in the
         notebook. This methods provides a unique identifier.
```
Comment (etejedor) [inline bindings/pyroot/JupyROOT/helpers/utils.py]: Can you extend the documentation of `_getUID` to explain why we need to create the identifier like this (i.e. explain JupyterLab case)?

**Comment by philipleindecker** [comment]
> Congrats @philipleindecker !
> I assume you checked that your changes also work with the classic Jupyter, right?

Yes I checked. It still works.

File: bindings/pyroot/JupyROOT/helpers/utils.py (hunk 1 above)
Comment (philipleindecker) [reply bindings/pyroot/JupyROOT/helpers/utils.py]: I just tried it without the `v`. It still works so I will remove it.

**Comment by etejedor** [comment]
@phsft-bot build

**Pull request merged**